examples/textual_inversion/README_sdxl.md

```diff
@@ -0,0 +1,27 @@
+## Textual Inversion fine-tuning example for SDXL
+
+The `textual_inversion.py` do not support training stable-diffusion-XL as it has two text encoders, you can training SDXL by the following command:
```

Comment:
```suggestion
```

I don't think we need to mention this. We can just add a note about the SDXL variant in the README.md file.